pkg/features/kube_features.go

```diff
@@ -194,7 +197,7 @@ var defaultFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	MultiplePreemptions:                 {Default: true, PreRelease: featuregate.GA},
 	TopologyAwareScheduling:             {Default: false, PreRelease: featuregate.Alpha},
 	ConfigurableResourceTransformations: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
I lost this thread but why are we promoting WorkloadResource to stable and leaving transformations in Betw. They were part of the same KEP..

Reply:
I see. https://github.com/kubernetes-sigs/kueue/pull/4166#issuecomment-2640830796